subprojects/instant-execution/src/main/kotlin/org/gradle/instantexecution/serialization/Codec.kt

```diff
@@ -119,9 +124,30 @@ enum class PropertyKind {
 }
 
 
+sealed class IsolateOwner {
+
+    abstract val delegate: Any
+
+    fun <T> service(type: Class<T>): T =
```

Comment:
⭕️ I'd move the `service` method above the `delegate` because it motivates the need for it.